tf_encrypted/protocol/securenn/odd_tensor.py

```diff
@@ -70,7 +71,20 @@ def sample_uniform(self,
                            maxval: Optional[int] = None):
             assert minval is None
             assert maxval is None
-            return OddUniformTensor(shape=shape, seed=seed())
+
+            if secure_random.supports_seeded_randomness():
+                seed = secure_random.seed()
+                return OddUniformTensor(shape=shape, seed=seed)
+
+            elif secure_random.supports_secure_randomness():
```

Comment:
Not sure I get the point of having these two flags. They both come from the same check for the secure random module so they are both going to be false all the time.

Reply:
i basically just wanted to have an easy way to turn off seeding in upcoming benchmarks

Reply:
Ahh yes good idea!